great_expectations/rule_based_profiler/rule/rule_output.py

```diff
@@ -0,0 +1,49 @@
+from typing import List, Optional, Union
+
+from great_expectations.core import ExpectationConfiguration
+from great_expectations.core.batch import Batch, BatchRequestBase
+from great_expectations.rule_based_profiler.expectation_configuration_builder import (
+    ExpectationConfigurationBuilder,
+)
+from great_expectations.rule_based_profiler.types import Domain, RuleState
+
+
+class RuleOutput:
+    """
+    RuleOutput provides methods for extracting useful information from RuleState using directives and application logic.
+    """
```

Comment:
A clarification question for my understanding : "is this where we will eventually be having a `metrics(self)` method?

Reply:
@Shinnnyshinshin Yes!  Maybe somewhat different signature, but yes!